cohesive_energy.py

```diff
@@ -0,0 +1,28 @@
+from ase.md.velocitydistribution import (MaxwellBoltzmannDistribution,Stationary,ZeroRotation)
+from ase.md.verlet import VelocityVerlet
+from asap3 import Trajectory
+from ase import units
+from equilibriumCondition import equilibiriumCheck
+
+
+
+def cohesive_energy(atoms,iterations):
+    """The cohesive_energy function takes argument atoms, that is the simulated material,
+    and makes another simulation for the material at temperature 0 K.
+    It the looks extracts the potential energy which is the cohesive energy for the material"""
+    MaxwellBoltzmannDistribution(atoms, temperature_K=0)
+    Stationary(atoms)
+    ZeroRotation(atoms)
+    coh_dyn = VelocityVerlet(atoms, 5 * units.fs)
```

Comment:
Could this use the timestep from the config file and also name the .traj-file to somthing like <symbol>_coh_E.traj so processes don't risk overwriting things that are in use. It would also be nice if this file also ended up in the output directory.

Reply:
Fixed!